routes/application.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const applicationController = require("../controller/ApplicationsController");
+
+router.get(
+  "/:organizationId/applications",
```

Comment:
Your route should be just `/` not `/:organizationId/applications`. Check docs to confirm